src/resources/views/crud/form_content.blade.php

```diff
@@ -190,15 +190,27 @@ function preventUnload(event) {
                         $('[name="' + normalizedProperty + '[]"]') :
                         $('[name="' + normalizedProperty + '"]'),
                         container = field.parent('.form-group');
-          
+
+            // Check if phone field
+            if(field.length > 0 && container.length == 0) {
+                if(field.parent().hasClass('iti')) {
+                    container = field.parent().parent('.form-group');
+                }
+            }
+
             // iterate the inputs to add invalid classes to fields and red text to the field container.
-            container.children('input, textarea, select').each(function() {
+            container.find('input, textarea, select').each(function() {
                 let containerField = $(this);
                 // add the invalida class to the field.
                 containerField.addClass('is-invalid');
                 // get field container
                 let container = containerField.parent('.form-group');
 
+                // Check if phone field
```

Comment:
Having changes on this file for this particular field (repeatable it's an exception because it's a container of fields), triggers my alert bells. 

Can't we change the field structure to match the rule ? Making the `iti` identifier a span or something that does not break "child->parent" container/field relation ?

Reply:
Ah good catch Pedro. I didn't see this has changes in `form_content`. Totally - this new field type should not change `form_content` at all.

Great catch 👏👏👏